experimenter/tests/integration/nimbus/conftest.py

```diff
@@ -57,6 +57,15 @@
     BaseExperimentApplications.DEMO_APP.value: KINTO_COLLECTION_WEB,
 }
 
+APPLICATION_SELECT_VALUE = {
+    BaseExperimentApplications.FIREFOX_DESKTOP.value: "firefox-desktop",
+    BaseExperimentApplications.FIREFOX_FENIX.value: "fenix",
+    BaseExperimentApplications.FIREFOX_IOS.value: "ios",
+    BaseExperimentApplications.FOCUS_ANDROID.value: "focus-android",
+    BaseExperimentApplications.FOCUS_IOS.value: "focus-ios",
+    BaseExperimentApplications.DEMO_APP.value: "demo-app",
+}
```

Comment:
we are missing fxa, monitor

Reply:
I know and I was gonna add them but then I realized they're also missing 'no-feature', which the tests depend on, so we'll have to do it in a separate PR.